python/ray/util/client/server/server.py

```diff
@@ -477,7 +477,7 @@ def _schedule_named_actor(self,
                               task: ray_client_pb2.ClientTask,
                               context=None) -> ray_client_pb2.ClientTaskTicket:
         assert len(task.payload_id) == 0
-        actor = ray.get_actor(task.name)
+        actor = ray.get_actor(task.name, task.namespace or None)
```

Comment:
nit: Can you mention that this `or` here is to convert an empty string -> `None`?

Reply:
Done